File: gasolina/gasolina_back/gasoline/models/venta.py
from django.db import models

from gasoline.models import Bomba, TipoCombustible


class Venta(models.Model):
    nombre_factura = models.CharField(max_length=100)
    nit = models.CharField(max_length=20)
    cliente = models.CharField(max_length=100)
    correo = models.EmailField(max_length=100)
    monto = models.FloatField()
    precio_actual = models.FloatField()
    cantidad_litros = models.FloatField()
    bomba = models.ForeignKey(Bomba, on_delete=models.CASCADE, related_name='bomba')
    fecha_hora = models.DateTimeField(auto_now_add=True)
    tipo_producto = models.ForeignKey(TipoCombustible, on_delete=models.CASCADE, related_name='tipo_producto')

File: gasolina/gasolina_front/src/models/Venta.ts
export interface Venta {
    id?: number;
    nombre_factura: string;
    nit: string;
    cliente: number;
    correo: string;
    monto: string;
    precio_actual: string;
    cantidad_litros: string;
    bomba: number;
    fecha_hora: string;
    tipo_producto : number;
}
